policy/modules/system/systemd.te

```diff
@@ -604,6 +604,8 @@ kernel_dontaudit_search_unlabeled(systemd_generator_t)
 # vmware_vsock
 kernel_request_load_module(systemd_generator_t)
 
+init_getattr_generic_units_files(systemd_generator_t)
```

Comment:
This shouldn't be necessary due to the below `init_read_generic_units_files(systemd_generator_t)`